android/brave_java_resources.gni

```diff
@@ -742,8 +745,10 @@ brave_java_resources = [
   "java/res/drawable/wallet_top_banner_bg.xml",
   "java/res/drawable/wallet_verify_button.xml",
   "java/res/drawable/warning_circle_filled.xml",
+  "java/res/drawable/warning_circle_filled_theme_color.xml",
   "java/res/drawable/web_notification_small_icon_background.xml",
   "java/res/drawable/white_rounded_holo_button_thin.xml",
+  "java/res/font/roboto_mono_regular.ttf",
```

Comment:
@aguscruiz do we need new font for the changes ?

Reply:
@deeppandya 
 I believe, we need. I could specify Roboto regular, it present on devcies, but we need `mono` which means the width of all chars is equal. I couldn't find the way to enable roboto mono without embeding.
 
 I had to take it from https://fonts.google.com/specimen/Roboto+Mono